pkg/minikube/node/start.go

```diff
@@ -96,6 +96,7 @@ func Start(starter Starter, apiServer bool) (*kubeconfig.Settings, error) {
 		return nil, err
 	}
 	if stopk8s {
+		out.Step(style.Docker, "Preparing Docker 20.10.12 ...")
```

Comment:
This needs to be dynamic, and use the same code as when running _with_ Kubernetes